apollo-federation/src/utils/logging.rs

```diff
@@ -50,3 +50,23 @@ macro_rules! snapshot {
 }
 
 pub(crate) use snapshot;
+
+#[allow(unreachable_pub)] // suppress warning: tracing utility
+pub fn make_string<T>(
```

Comment:
I couldn't find a utility function like this on the web. But, I could've missed something better.